docs/book/customers/customer_and_shopuser.rst

```diff
@@ -28,8 +28,7 @@ As usual, use a factory. The only required field for the Customer entity is ``em
 
    $this->container->get('sylius.repository.customer')->add($customer);
 
-The Customer entity can of course hold other information besides an email, it can be for instance
-``billingAddress`` and ``shippingAddress``, ``firstName``, ``lastName`` or ``birthday``.
+The Customer entity can of course hold other information besides an email, it can be for instance ``firstName``, ``lastName`` or ``birthday``.
```

Comment:
Customer has `defaultAddress` and `addresses` collection, maybe we should mention them here, just to keep some information about customer addresses? :)

Reply:
This brings up an overall problem of the documentation. The information is spread over all the components and bundles. In my opinion the information that there are other fields has no real value to the reader at this point. If you mention the `addresses` collection you should also mention the AddressBook and that the addresses only shown to the customer in his / her profile but not in the admin panel.